cadnano/part/nucleicacidpart.py

```diff
@@ -2783,8 +2783,6 @@ def batchCreateVirtualHelices(self,
         assert z_list is None or len(z_list) == len(x_list)
         assert length is None or isinstance(length, (list, tuple))
         assert length is None or len(length) == len(x_list)
-        assert id_num is None or isinstance(id_num, (list, tuple))
```

Comment:
What was wrong with these two assertions?

Reply:
The variable `id_num` does not exist and causes a `NameError`

Reply:
Done